operations/helm/charts/mimir-distributed/values.yaml

```
@@ -2636,6 +2636,19 @@ metaMonitoring:
     # Note that you need to configure serviceMonitor in order to have some metrics available.
     enabled: false
 
+    # -- Controls the image repository and tag for config-reloader and grafana-agent containers in the meta-monitoring
+    # statefulSet and daemonSet created by the grafana-agent-operator. You can define one or both sections under imageRepo.
+    # If a section is defined, you must pass repo, image and tag attributes.
```

nit: the docs are using `field` instead of attribute in a few places. But I think it should be `key` according to what the YAML docs use https://www.yaml.info/learn/index.html

```suggestion
    # If a section is defined, you must pass repo, image and tag keys.
```